klass-api/src/test/java/no/ssb/klass/api/migration/dataintegrity/KlassApiClassificationCodesTest.java

```diff
@@ -0,0 +1,180 @@
+package no.ssb.klass.api.migration.dataintegrity;
+
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.time.Instant;
+import java.time.LocalDate;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+import static no.ssb.klass.api.migration.MigrationTestConstants.*;
+import static no.ssb.klass.api.migration.MigrationTestUtils.*;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assumptions.assumeTrue;
+
+public class KlassApiClassificationCodesTest extends AbstractKlassApiDataIntegrityTest {
+
+    static String dateFromToMax;
+
+    static LocalDate dateOne;
+    static LocalDate dateTwo;
+    static String dateFromInRangeString ;
+    static String dateToInRangeString;
+
+    static Integer randomId;
+
+    static Map<String, Object> paramsDate = new HashMap<>();
+    static Map<String, Object> paramsDateInRange = new HashMap<>();
+
+    Response sourceResponse;
+    Response targetResponse;
+
+    @BeforeAll
+    static void beforeAllCodes() {
+         dateFromToMax = generateRandomDate(
+                 LocalDate.of(1900, 1, 1),
+                 LocalDate.of(2030, 12, 31)).format(formatter);
+         dateOne = generateRandomDate(
+                 LocalDate.of(1900, 1, 1),
+                 LocalDate.of(2030, 12, 31)
+         );
+         dateTwo = generateRandomDate(
+                 LocalDate.of(1900, 1, 1),
+                 LocalDate.of(2030, 12, 31)
+         );
+
+         dateFromInRangeString = dateOne.isBefore(dateTwo) ? dateOne.format(formatter) : dateTwo.format(formatter);
+         dateToInRangeString = dateOne.isAfter(dateTwo) ? dateOne.format(formatter) : dateTwo.format(formatter);
+
+         randomId = generateRandomId(150);
+
+         paramsDate.put(RANGE_FROM, dateFromToMax);
+         paramsDateInRange.put(RANGE_FROM, dateFromInRangeString);
+         paramsDateInRange.put(RANGE_TO, dateToInRangeString);
+
+        RestAssured
+                .given()
+                .baseUri("http://localhost:8081")
+                .basePath("/api/klass/v1")
+                .get("/classifications")
+                .then()
+                .statusCode(200);
+
+        RestAssured
+                .given()
+                .baseUri("http://localhost:8080")
+                .basePath("/api/klass/v1")
+                .get("/classifications")
+                .then()
+                .statusCode(200);
```

Comment:
It looks like we're making some assumptions about where `source` and `target` are hosted?